packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -366,6 +366,107 @@ export = {
       },
     },
 
+    'with client\'s TLS certificate from SDS': {
```

Comment:
Let's just use double quotes for strings that include `'`:

```suggestion
    "with client's TLS certificate from SDS": {
```

Reply:
Addressed